videojs-contrib-dash.html

```diff
@@ -17,12 +17,20 @@
   <script src="js/docs-themes.js" type="text/javascript"></script>
   <script type="text/javascript">set_style();</script>
 
+  <!-- Chromecast pre-load -->
+  <head data-cast-api-enabled="true"> <!-- allow chromecast extention -->
+  <script src="http://www.gstatic.com/cv/js/sender/v1/cast_sender.js"></script>
```

Comment:
We should omit the scheme from this url so it matches the page, otherwise this'll create mixed-content warnings on https.